### ISSUE TITLE
[page-contracts]: wrong default parameter

### ISSUE THREAD

**Issue by akru** [polkadot-js/apps#3036]
**Traceback**

```
polkadotjs.809370bb.js:1 2020-06-24 12:15:57        RPC-CORE: call(callRequest: ContractCallRequest, at?: BlockHash): ContractExecResult:: -32602: `1000000000000` > u32::max_value(), the max block number is u32.
u @ polkadotjs.809370bb.js:1
polkadotjs.809370bb.js:1 Uncaught (in promise) Error: -32602: `1000000000000` > u32::max_value(), the max block number is u32.
    at e.default._checkError (polkadotjs.809370bb.js:1)
    at e.default.decodeResponse (polkadotjs.809370bb.js:1)
    at Y.value (polkadotjs.809370bb.js:1)
    at WebSocket.value (polkadotjs.809370bb.js:1)
```

**Screenshot**
![wrong_gas_allowed](https://user-images.githubusercontent.com/786394/85528535-add15600-b614-11ea-83dd-4fb115ddc0e3.png)

**Description**

* Substrate: `2.0.0-rc3`
* PolkadotJS/Apps: `0.47.0-beta.3`

Unfortunately `call` contract for [Smart contracts workshop](https://substrate.dev/substrate-contracts-workshop/#/0/calling-your-contract) isn't working on the latest substrate - polkadot apps.

But, when I decrease default `maximum gas allowed` value from `1000000` to `1000` it works well.

**Comment by jacogr** [comment]
This is a bug on the Substrate RPC, see https://github.com/polkadot-js/apps/pull/2743#issuecomment-628248018

(It is fixed in latest Substrate master)

**Comment by akru** [comment]
Oh, thank you @jacogr, that's mean it's not included in `rc3` release, right?

**Comment by jacogr** [comment]
No, rc3 was replaced by rc4 (which is more than 2 weeks old by now), it was recently fixed, I believe last week - so rc5 seems to be due his week, which means it will include the fix.

**Comment by akru** [comment]
Thank you @jacogr, looking forward to `rc5` in crates!

**Comment by akru** [comment]
Fixed in substrate (2.0.0-rc4](https://github.com/paritytech/substrate/releases/tag/v2.0.0-rc4)

**Comment by jacogr** [comment]
Brilliant! Seems I got the rc4/rc5 a bit wrong... Sorry about that. (Sometimes get a bit lost at the pace of updates, or lack thereof for releases)
